core/src/components/datetime/readme.md

```diff
@@ -91,7 +91,7 @@ Ionic uses the [ISO 8601 datetime format](https://www.w3.org/TR/NOTE-datetime)
 for its value. The value is simply a string, rather than using JavaScript's
 `Date` object. Additionally, when using the ISO datetime format, it makes it
 easier to serialize and pass within JSON objects, and sending databases a
```

Comment:
I know this wasn't a change made by you, but should it be

```suggestion
easier to serialize and pass within JSON objects, and send databases a
```

instead?

Reply:
Yeah, that sounds better. Maybe the whole sentence needs a rewrite... What do you think of: 

> Additionally, the ISO datetime format is easier to serialize and pass within JSON objects. As it is a standardized format, it can also easily be sent to and parsed by databases.

Reply:
maybe even simpler?
```suggestion
Using the ISO datetime format makes it easy to serialize and parse within JSON objects and databases.
```

Reply:
Looks good to me, see https://github.com/ionic-team/ionic/pull/19316/commits/6393b996c708a6cc3c7ef0c8b5140bda5c92e720